{"uuid":"fb50baea-6478-42cf-9120-f62b70017f52","historyId":"a1e7cca7aef819cd5be44e7e0dc680a3","fullName":"com.google.testgooglecalculator.MainGooglePageTest.calculationShouldEqualTo1","labels":[{"name":"package","value":"com.google.testgooglecalculator.MainGooglePageTest"},{"name":"testClass","value":"com.google.testgooglecalculator.MainGooglePageTest"},{"name":"testMethod","value":"calculationShouldEqualTo1"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"testGoogleCalculator"},{"name":"subSuite","value":"com.google.testgooglecalculator.MainGooglePageTest"},{"name":"host","value":"MacBook-Air-Konstantin-2.local"},{"name":"thread","value":"17322@MacBook-Air-Konstantin-2.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"calculationShouldEqualTo1","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Результат расчета не соответствует ожидаемому expected [1] but found [3.1]","trace":"java.lang.AssertionError: Результат расчета не соответствует ожидаемому expected [1] but found [3.1]\n\tat org.testng.Assert.fail(Assert.java:99)\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\n\tat org.testng.Assert.assertEquals(Assert.java:122)\n\tat org.testng.Assert.assertEquals(Assert.java:629)\n\tat com.google.testgooglecalculator.MainGooglePageTest.calculationShouldEqualTo1(MainGooglePageTest.java:58)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.util.ArrayList.forEach(ArrayList.java:1259)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\n\tat org.testng.TestRunner.run(TestRunner.java:596)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\n\tat org.testng.TestNG.run(TestNG.java:1031)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1644699256540,"stop":1644699258235}